tasks/setup_static.yml

```diff
@@ -0,0 +1,42 @@
+---
+- name: Create static dirs
+  ansible.builtin.file:
+    state: directory
+    mode: '0755'
+    path: "{{ subdomains_static_path }}/static-{{ subdomain.name }}"
+
+- name: Move all contents from static to parent directory
+  ansible.builtin.command: cp -r {{ subdomains_static_dir }}/. {{ subdomains_static_path }}/static-{{
```

Comment:
any reason to not use copy remote_src=true?

Reply:
I think the '-r' is the problem